### PR TITLE
news: Add hoki news page.

### PR DESCRIPTION
**Pull request by MagneFire**
This adds a news post for the Fossil Gen 6 (hoki) port.

It contains:
- The challenges with obtaining the kernel sources.
- The quirks with getting the port to work.

This depends on an actual build, so basically: https://github.com/AsteroidOS/meta-smartwatch/pull/183

**Date needs to be adjusted.**

### CODE REVIEW

File: pages/news/hokiport.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+author: Darrel Griët (MagneFire)
+---
+
+AsteroidOS has been ported to the Fossil Gen 6!
+
+This port is quite unique as it's the first Snapdragon Wear 4100 watch to gain AsteroidOS support.
+
+The port isn't perfect out of the box. Check our [GitHub repository](https://github.com/AsteroidOS/meta-smartwatch/issues?q=is%3Aissue+is%3Aopen+label%3Ahoki) for the current issues..
+
+At the time of writing many of the supported watches aren't available new. This brought up a desire to try and find a newer watch to port. Gaining the knowledge from this could help future porters as well.
```
Comment (FlorentRevest) [inline pages/news/hokiport.md:17]: How about a rewording of this paragraph like ~~~this:

Most of the currently supported watches can't be bought new. This brought up the desire to port AsteroidOS to newer watches. Breaking new grounds with this sort of ports also helps us gain knowledge useful for future ports.

I find it makes it a bit easier to read/clearer.

File: pages/news/hokiport.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Getting the kernel to compile was a bit challenging as the vendor kernel doesn't actually provide a [device tree](https://github.com/fossil-engineering/kernel-msm-fossil-cw/blob/fossil-android-msm-hoki-lw1.2-4.14/arch/arm64/boot/dts/qcom/Makefile#L2). The device tree was recreated by dumping the flash memory using [this](https://forum.xda-developers.com/t/twrp-twrp-for-fossil-gen-6-aka-hoki.4474863/) TWRP image followed by decompiling the Device tree Blob and packaging that with the kernel source. There was also a function missing. Once this was fixed the kernel started booting!
+
+Relatively few other patches were required to get the AsteroidOS UI up and running. It mostly consisted of adapting `beluga` specific changes such that they would work with `hoki`.
+
+Obviously this port isn't perfect so follow our [issues]((https://github.com/AsteroidOS/meta-smartwatch/issues?q=is%3Aissue+is%3Aopen+label%3Ahoki)) and check the website for the latest support status updates!
+
+# Thank you
+
+This port wouldn't be possible with the help from the following individuals:
```
Comment (FlorentRevest) [inline pages/news/hokiport.md:46]: without* ... :)

File: pages/news/hokiport.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- dodoradio with helping write formal e-mails to Fossil Group.
```
Comment (FlorentRevest) [inline pages/news/hokiport.md:47]: s/with helping/who helped/

File: pages/news/hokiport.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Charlotte from Fossil Group. Helping raise the issue internally.
```
Comment (FlorentRevest) [inline pages/news/hokiport.md:48]: s/. Helping/who helped/

File: pages/news/hokiport.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+But it took a while for Fossil Group to actually release the kernel sources.
+While Fossil Group mentioned things like: "Fossil supports open source and complies with open source licenses." and "Our Wear OS by Google based smartwatches are using the Linux Kernel which is licensed under GPL." while also linking Google kernel sources they couldn't help directly when it came to the Fossil Gen 6.
+As I contacted Fossil on separate occasions about this I was ensured that this issue was raised internally and I would get an update by e-mail. It took a couple of tries but eventually a case number was created and the conversation continued via e-mail as opposed to their online chat.
+After some back and forth via e-mail kindly asking for the kernel sources they eventually escalated this to the engineering team. This was around the end of 2022. I decided to give Fossil Group some time to get the kernel sources ready. And contacted them again at the beginning of April. At which point they responded to wait a little longer. Two days later the kernel sources were [released](https://github.com/fossil-engineering/kernel-msm-fossil-cw)!
```
Comment (FlorentRevest) [inline pages/news/hokiport.md:29]: Maybe we can shorten this story a little bit ? Just explaining that there were a bunch of back and forths and that it took a while without necessarily giving a step by step storyline ?

File: pages/news/hokiport.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+This turned out to make an initial port relatively straightforward with some exceptions of course 😅.
+
+Getting the kernel to compile was a bit challenging as the vendor kernel doesn't actually provide a [device tree](https://github.com/fossil-engineering/kernel-msm-fossil-cw/blob/fossil-android-msm-hoki-lw1.2-4.14/arch/arm64/boot/dts/qcom/Makefile#L2). The device tree was recreated by dumping the flash memory using [this](https://forum.xda-developers.com/t/twrp-twrp-for-fossil-gen-6-aka-hoki.4474863/) TWRP image followed by decompiling the Device tree Blob and packaging that with the kernel source. There was also a function missing. Once this was fixed the kernel started booting!
+
+Relatively few other patches were required to get the AsteroidOS UI up and running. It mostly consisted of adapting `beluga` specific changes such that they would work with `hoki`.
+
+Obviously this port isn't perfect so follow our [issues]((https://github.com/AsteroidOS/meta-smartwatch/issues?q=is%3Aissue+is%3Aopen+label%3Ahoki)) and check the website for the latest support status updates!
```
Comment (FlorentRevest) [inline pages/news/hokiport.md:42]: This is the same as the third paragraph. Maybe we should only keep one or the other ?

**Comment by MagneFire** [comment]
All feedback has been addressed. The story about obtaining the kernel sources has been reduced and the minute steps have been removed as well.

**Review by FlorentRevest — APPROVED** [review]

File: pages/news/hokiport.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+author: Darrel Griët (MagneFire)
+---
+
+AsteroidOS has been ported to the Fossil Gen 6!
+
+This port is quite unique as it's the first Snapdragon Wear 4100 watch to gain AsteroidOS support.
+
+The port isn't perfect out of the box. Check our [GitHub repository](https://github.com/AsteroidOS/meta-smartwatch/issues?q=is%3Aissue+is%3Aopen+label%3Ahoki) for the current issues.
+
+Most of the currently supported watches can't be bought new. This brought up the desire to port AsteroidOS to newer watches. Breaking new grounds with this sort of ports also helps us gain knowledge useful for future ports.
```
Comment (eLtMosen) [inline pages/news/hokiport.md:17]: Maybe a bit better flowing would be
`Breaking new grounds with this sort of ports helps us to gain useful knowledge for future ports.`

File: pages/news/hokiport.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+I didn't think this would be particularly hard since all previous Fossil and Skagen watches have the kernel sources hosted by [Google](https://android.googlesource.com/kernel/msm/+refs).
+However it took Fossil Group some time to release the kernel sources for the Fossil Gen 6.
+After a couple of back and forths with Fossil Group, an issue was raised internally. Following this, it took about four months for them to release the kernel [sources](https://github.com/fossil-engineering/kernel-msm-fossil-cw).
+
+This would make it possible to start a new port.
+
+# The port
+
+The watch originally launched with Android 9 based WearOS 2.0 which someone on XDA has posted for [download](https://forum.xda-developers.com/t/need-fossil-gen-6-ota-url.4387157/post-86792599).
+This turned out to make an initial port relatively straightforward with some exceptions of course 😅.
```
Comment (eLtMosen) [inline pages/news/hokiport.md:35]: `straightforward`. `With some`
New sentence after straightforward?

**Review by eLtMosen — COMMENTED** [review]
Sorry for chiming in so late.
But following is what i found only after the 3rd read now.

File: pages/news/hokiport.md (hunk 7 above)
Comment (MagneFire) [reply pages/news/hokiport.md]: This was also copied.

**Review by beroset — APPROVED** [review]
Nice!  I am looking forward to trying this port.

**Pull request merged**